src/Components/Alert/Components/Filters/ArtistSeries.tsx

```diff
@@ -0,0 +1,100 @@
+import { FC, Suspense } from "react"
+import { QuickMultipleSelectAlertFilter } from "Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter"
+import { ArtistSeriesOptionsQuery } from "__generated__/ArtistSeriesOptionsQuery.graphql"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import {
+  Flex,
+  Separator,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+} from "@artsy/palette"
+import { compact, times } from "lodash"
+
+const artistSeriesOptionsQuery = graphql`
+  query ArtistSeriesOptionsQuery($input: FilterArtworksInput!) {
+    artworksConnection(size: 0, input: $input) {
+      aggregations {
+        counts {
+          count
+          name
+          value
+        }
+        slice
+      }
+    }
+  }
+`
+
+export const ArtistSeries: FC = () => {
+  const { state } = useAlertContext()
+  const artistIDs = state?.criteria?.artistIDs
+
+  const data = useLazyLoadQuery<ArtistSeriesOptionsQuery>(
+    artistSeriesOptionsQuery,
+    {
+      input: { artistIDs, aggregations: ["ARTIST_SERIES"] },
+    }
+  )
+  const artistSeries = (data.artworksConnection?.aggregations || []).find(
+    aggs => aggs?.slice === "ARTIST_SERIES"
+  )
+
+  const options = compact(
+    (artistSeries?.counts ?? []).map(count => {
+      if (!count) return null
+
+      return {
+        name: count.name,
+        value: count.value,
+      }
+    })
+  )
+
+  if (!options.length) return null
+
+  return (
+    <>
+      <Separator my={2} />
+
+      <QuickMultipleSelectAlertFilter
+        label="Artist Series"
+        // description="Optionally narrow down your alert to one or more of these series within this artist’s body of work"
+        criteriaKey="artistSeriesIDs"
+        options={options}
+        truncate={8}
+      />
+    </>
+  )
+}
+
+export const ArtistSeriesQueryRenderer: React.FC = props => {
```

Comment:
I'm not up to date on FE patterns re: `Suspense`, so I was expecting to see `SystemQueryRenderer` being used here, but it's not - is this actually a query renderer?

Reply:
Fair question, I took my lead from [this very recent addition](https://github.com/artsy/force/blob/f2475786be4945796efbfec10487f537ba1bfda4/src/Components/Alert/Components/Form/SuggestedFilters.tsx#L137).